recipe/build.sh

```diff
@@ -86,6 +86,9 @@ if [[ "$target_platform" == linux-* || "$target_platform" == win-* ]]; then
   export LDFLAGS="$LDFLAGS -static-libstdc++ -static-libgcc"
 fi
 
+# explicitly set c99
+export CFLAGS="$CFLAGS -std=c99"
```

Comment:
Thanks @h-vetinari ! This looks already much better now :-) I took some look at the error message with the `for` loop and C99 construct. And from the relevant Makefile I found the right variable to set (see Makefile: https://github.com/bminor/binutils-gdb/blob/binutils-2_42/bfd/doc/local.mk#L85 )
```suggestion
export CFLAGS="$CFLAGS -std=c99"
export CFLAGS_FOR_BUILD="$CFLAGS"
```
Locally I still had some issues with cross compilation tests, but I think this was only as the test tried to run on the wrong architecture of my machine.